modules/simphera_instance/postgresql.tf

```diff
@@ -4,22 +4,34 @@ resource "azurerm_resource_group" "postgres" {
   tags     = var.tags
 }
 
+resource "random_password" "postgresql-rnd-pass" {
+  length           = 16
+  special          = true
+  override_special = "!#$%&*-_=+:?"
+}
+
+resource "azurerm_key_vault_secret" "postgresql-credentials" {
+  name         = "${var.name}-postgresqlcredentials"
+  value        = jsonencode({ "postgresql_username" : "dbuser", "postgresql_password" : random_password.postgresql-rnd-pass.result })
```

Comment:
In future versions of SIMPHERA we would like to enable the key rotation feature of the keyvault. Probably, we should already prepare the reference architecture for this. @mariogluhakovic can you do a little research what needs to be done  at the Terraform side to enable secret rotation?

Reply:
As we discussed, we skip the considerations for key rotation for now